sdk/communication/communication-phone-numbers/src/phoneNumbersClient.ts

```diff
@@ -110,9 +109,16 @@ export class PhoneNumbersClient {
       },
     };
 
+    this.client = new PhoneNumbersGeneratedClient(url, {
```

Comment:
A couple lines above, the code to handle `userAgentOptions` should be removed. https://github.com/Azure/autorest.typescript/wiki/%60core-http%60-dependency-migration-to-%60core-client%60-%60core-rest-pipeline%60#remove-the-code-to-handle-useragentoptions

Reply:
Thanks, I wasn't aware of that guide. I will update accordingly!